R/metadata.R

```diff
@@ -10,28 +11,33 @@
 #'
 #' @examples
 #' get_plotting_metadata("Malawi")
-get_plotting_metadata <- function(country) {
+#' get_plotting_metadata()
+get_plotting_metadata <- function(country = "default") {
```

Comment:
What's the use case in which country name won't be known when calling this function?

Reply:
Good point - there isn't really a use case for doing that. It will always be known - I will take that out.